README.md

```diff
@@ -2,22 +2,30 @@
 [![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-mscalendar/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-mscalendar)
 [![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-mscalendar/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-mscalendar)
 
-# [Help Wanted](https://github.com/mattermost/mattermost-plugin-mscalendar/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22up+for+grabs%22+label%3A%22help+wanted%22+sort%3Aupdated-desc)
+## Help Wanted tickets can be found [here](https://github.com/mattermost/mattermost-plugin-mscalendar/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22up+for+grabs%22+label%3A%22help+wanted%22+sort%3Aupdated-desc)
+
+## Table of Contents
+1. [Overview](#overview)
+2. [Features](#features)
+3. [Configuration](#configuration)
+
+## Overview
 
 This plugin supports a two-way integration between Mattermost and Microsoft
 Outlook Calendar. For a stable production release, please download the latest
 version [in the Releases
 tab](https://github.com/mattermost/mattermost-plugin-mscalendar/releases) and
```

Comment:
nit: Since there are no releases yet and no stable production release... should we put this? Wouldn't it be better at the moment put something like this is still WIP? Not sure though.

Reply:
I feel like we could leave it since it is not a broken link and won't require tracking to remember adding after we do a first release.